runtime/near-runtime-fees/Cargo.toml

```diff
@@ -14,6 +14,7 @@ Fees applied to near runtime encapsulated in a separate crate. Might merge it la
 
 [dependencies]
 serde = { version = "1", features = ["derive"] }
+near-primitives = { path = "../../core/primitives" }
```

Comment:
```suggestion
near-primitives = { path = "../../core/primitives", version = "0.1.0" }
```

Reply:
looking over this crate, it just re-exports the near_primitives so perhaps it's not needed anymore.  Do you think I should just refactor all deps to use `near_primitives::runtime::fees`?

Reply:
Just a reminder: we cannot have near-primitives dependency in near-runtime-fees (see my other comment)